knowledge-base/common-layout-components-bootstrap-alternatives.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Layout Components in Telerik UI for Blazor
```

Comment:
I guess your purpose is to guide the developer in choosing the most suitable component, so you can state this directly.
```suggestion
title: Choosing the Right Layout Components for Your App
```

Reply:
Actually we refrain from consulting customers what to use. I also aimed for a shorter title that starts with a meaningful and relevant word.